index.bs

```diff
@@ -4295,8 +4319,8 @@ For example:
 
```

Comment:
Above this there's this example:

>When authenticating for the first time on a particular [client device](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1901.html#client-device), a [roaming authenticator](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1901.html#roaming-authenticators) is typically needed since the user doesn’t yet have a [platform credential](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1901.html#platform-credential) on that [client device](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1901.html#client-device).

which is probably no longer accurate with synced passkeys.

Reply:
Yes, I think this whole section needs some updating, but I was only trying to address changes specific to the new method.

Reply:
I'll address this in a separate PR (issue: #1912)